packages/jupyter-ai/src/components/chat-messages/chat-message-delete.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { IconButton, SxProps } from '@mui/material';
+import { Close } from '@mui/icons-material';
+
+import { AiService } from '../../handler';
+import { ChatHandler } from '../../chat_handler';
+
+type DeleteButtonProps = {
+  message: AiService.ChatMessage;
+  chatHandler: ChatHandler;
+  sx?: SxProps;
+};
+
+export function ChatMessageDelete(props: DeleteButtonProps): JSX.Element {
+  const request: AiService.ClearRequest = {
+    type: 'clear',
+    target: props.message.id
+  };
+  return (
+    <IconButton
+      onClick={() => props.chatHandler.sendMessage(request)}
+      sx={props.sx}
+    >
```

Comment:
There should be a tooltip on this button that tells the user exactly what this button does. It's not immediately obvious that the button deletes that message & all future messages.

1. Can you use the custom `TooltippedIconButton` component available in Jupyter AI instead of `IconButton`? This adds some custom styles and stops the tooltip from appearing too far from the target.
    - Specifically: `packages/jupyter-ai/src/components/mui-extras/tooltipped-icon-button.tsx`.
2. The tooltip should read: "Delete this and all future messages".